.travis.yml

```diff
@@ -34,6 +34,7 @@ before_install:
     - travis_retry composer self-update
 
 install:
+    - wget https://github.com/puli/cli/releases/download/1.0.0-beta10/puli.phar && chmod +x puli.phar
```

Comment:
Ping @dbu 
We download puli.phar here. We do also require "puli/symfony-bundle"